src/compiler/CodeGen.hpp

```diff
@@ -1,40 +1,44 @@
 #pragma once
 
-#include <vector>
+#include <binaryen-c.h>
 #include <deque>
-#include <string>
-#include <map>
+#include <filesystem>
 #include <fstream>
 #include <iostream>
+#include <map>
 #include <memory>
-#include <filesystem>
+#include <string>
+#include <vector>
+
+#include "../lexer/Lexemes.hpp"
 #include "../parser/ast/ASTNode.hpp"
 #include "../parser/ast/BinaryOperationNode.hpp"
-#include "../parser/ast/UnaryOperationNode.hpp"
 #include "../parser/ast/LiteralNode.hpp"
 #include "../parser/ast/SourceNode.hpp"
-#include "../lexer/Lexemes.hpp"
+#include "../parser/ast/UnaryOperationNode.hpp"
 #include "StandardLibrary.hpp"
-#include <binaryen-c.h>
 
 using namespace std;
 
 namespace Theta {
     class CodeGen {
-        public:
-            // using GenerateResult = std::variant<BinaryenExpressionRef, BinaryenLiteral, int>;
+    public:
+        // using GenerateResult = std::variant<BinaryenExpressionRef, BinaryenLiteral, int>;
 
-            static BinaryenModuleRef generateWasmFromAST(shared_ptr<ASTNode> ast);
-            static BinaryenExpressionRef generate(shared_ptr<ASTNode> node, BinaryenModuleRef &module);
-            static BinaryenExpressionRef generateBinaryOperation(shared_ptr<BinaryOperationNode> node, BinaryenModuleRef &module);
-            static BinaryenExpressionRef generateUnaryOperation(shared_ptr<UnaryOperationNode> node, BinaryenModuleRef &module);
-            static BinaryenExpressionRef generateNumberLiteral(shared_ptr<LiteralNode> node, BinaryenModuleRef &module);
-            static BinaryenExpressionRef generateStringLiteral(shared_ptr<LiteralNode> node, BinaryenModuleRef &module);
-            static BinaryenExpressionRef generateBooleanLiteral(shared_ptr<LiteralNode> node, BinaryenModuleRef &module);
-            static BinaryenExpressionRef generateExponentOperation(shared_ptr<BinaryOperationNode> node, BinaryenModuleRef &module);
-            static void generateSource(shared_ptr<SourceNode> node, BinaryenModuleRef &module);
+        static BinaryenModuleRef generateWasmFromAST(shared_ptr<ASTNode> ast);
+        static BinaryenExpressionRef generate(shared_ptr<ASTNode> node, BinaryenModuleRef &module);
+        static BinaryenExpressionRef generateBinaryOperation(shared_ptr<BinaryOperationNode> node,
```

Comment:
Is there a way to get this to format to:

```cpp
static BinaryenExpressionRef generateBinaryOperation(
    shared_ptr<BinaryOperationNode> node,
    BinaryenModuleRef &module
);
```

instead?

Reply:
Done